docs/wdc_oauth_tutorial.md

```diff
@@ -897,544 +960,598 @@ Web Data Connector `.js` library provides a default `init` function, so
+4.  Click the **Start Interactive Phase** button.
 
     The simulator opens a window and displays the UI for the Foursquare
     web data connector.
 
-    ![]({{ site.baseurl }}/assets/wdc_oauth_tutorial_run_interactive_phase_in_simulator.png)
+    ![]({{ site.baseurl }}/assets/wdc_oauth_tutorial_test_in_simulator.png)
 
-4.  If the page indicates that you're not logged in to Foursquare, click
-    the <span class="uicontrol">Connect to Foursquare</span> button and
+5.  If the page indicates that you're not logged in to Foursquare, click
+    the **Connect to Foursquare** button and
     log in.
 
-5.  Click the <span class="uicontrol">Get Venues I Like</span> button.
+6.  After you have logged in, click the **Get Venues I Like** button.
 
     The simulator closes the window that displayed the UI phase of the
     connector and returns to the main simulator window. At the bottom of
     the main simulator window, the simulator displays the column names
-    and the values for any venues that you've "Liked" on Foursquare.
+    and the types for the table schema you defined.  
 
-    ![]({{ site.baseurl }}/assets/wdc_tutorial_oauth_test_in_new_simulator.png)
+7.  Click **Fetch Table Data** to display the values for any venues that
+    you've "Liked" on Foursquare. 
 
-    Notice also that the simulator has filled the <span
-    class="uicontrol">Password</span> box with the authentication token.
+    ![]({{ site.baseurl }}/assets/wdc_oauth_tutorial_simulator_data.png)
+
+    Notice also that the simulator has filled the **Password** box with the authentication token.
     Remember that in your code, when you got the authentication token
     back from Foursquare, you set the `tableau.password` property to the
     token value. When you run the simulator, after the UI phase is done,
     the simulator displays the values of `tableau` object properties.
 
-Foursquare.html page {#html-page}
---------------------
 
-**Note**: This listing includes a reference to the `tableauwdc-1.1.1.js`
+
+---
+
+# Step 10 Test the Connector in Tableau 
+
+[(Back to top)](#top)
+
+In this final part of the tutorial, you'll use your connector in Tableau
+to display a map that includes locations that you've "Liked" in
+Foursquare.
+
+## Test in Tableau
+
+
+The OAuth Foursquare web data connector is complete! You can now test it
+in Tableau.
+
+1.  Make sure you have "Liked" at least a few venues on Foursquare, or
+    you won't have any data to view in Tableau.
+
+2.  Open Tableau.
+
+3.  In the **Connect** pane, under **To a Server**, 
+    choose **Web Data Connector**. You might need to choose **More...** to find it.
+
+ <!--    ![]({{ site.baseurl }}/assets/wdc_tutorial_oauth_viz.png) -->
+
+4.  Enter the URL of your connector
+    (`http://localhost:3333`).
+
+5.  Click the **Connect to Foursquare** button and sign in.
+
+6.  In the connector, click **Get Venues I Like**.
+
+    ![]({{ site.baseurl }}/assets/wdc_tutorial_oauth_basic_signin_ui.png)
```

Comment:
This image displays "you're signed in" and "you are not signed in" at the same time.